packages/ui-components/src/search-bar.ts

```diff
@@ -0,0 +1,16 @@
+import styles from "./search-bar.module.sass";
+import { Card, Icon } from "@blueprintjs/core";
+import hyper from "@macrostrat/hyper";
+
+const h = hyper.styled(styles);
+
+export function SearchBar({ onChange, placeholder = "Search..." }) {
+  return h(Card, { className: "search-bar" }, [
```

Comment:
I'm not particularly into the card-based styling of this search bar.

Could you use a `@blueprintjs/core` form component to avoid having to do so much styling yourself? If so, potentially you could avoid having to make your own shared component.